### PR TITLE
Add 7 with ESU & 8.1 as supported platform

### PR DESCRIPTION
**Pull request by BrunoDal**
According to this blogpost it's supported to run MDE P1 on Win7 with ESU & 8.1 : https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/microsoft-defender-for-endpoint-plan-1-now-included-in-m365-e3/ba-p/3060639

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/defender-endpoint-plan-1.md [hunk 1]
```diff
@@ -183,6 +183,8 @@ To learn more, see [Defender for Endpoint APIs](management-apis.md).
 
 Most organizations use various devices and operating systems. Currently, Defender for Endpoint Plan 1 supports the following operating systems:
 
+- Windows 7 (ESU Required)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/defender-endpoint-plan-1.md]:
```suggestion
- Windows 7 (ESU required)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@BrunoDal Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @BrunoDal @yogkumgit @JohanFreelancer9

**Pull request merged**